rfc/README.md

```diff
@@ -29,10 +29,11 @@ Here are the direct links to open, closed, or all RFC pull requests:
 
 The workflow to add a new RFC works as follows:
 
-1. Copy the template and increase the running counter:
+1. Create a new RFC directory (increase the counter) and copy the template:
    ```bash
-   # Assumption: the last proposal is 041-proposal.md
-   cp 000-TEMPLATE.md 042-proposal-name.md
+   # Assumption: the last proposal is 041-proposal
+   mkdir 042-proposal-name
+   cp 000-TEMPLATE.md 042-proposal-name/README.md
```

Comment:
Any reason you didn't move the template to `000-TEMPLATE/README.md`?

Reply:
No, would you prefer this? (I don't have a strong opinion.)

Reply:
I applied your suggestion.